sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,906 @@
+push their transactions through.
+
+### Transaction Fees
+
+The user pays the transaction fee for their sBTC peg-in transaction.  Stackers
+do not pay a STX transaction fee, but instead require miners to include their
+peg-in authorization transaction in the next block they sign.
+
+### Two-phase Peg-In
+
+Users may not be able to send a Bitcoin transaction with an `OP_RETURN` output.
+For example, this is difficult to do or is unsupported in some Bitcoin wallets,
+especially custodial wallets.  To support these users, a 3rd party could simply
+send peg-in transaction on their behalf.  The user would simply pay the 3rd
+party the requisite Bitcoin to peg-in, and the 3rd party would generate and send
+the peg-in transaction for them.
+
+## Peg-Out
+
+Pegging out is the act of redeeming sBTC for BTC (Figure 2).
+To redeem sBTC for BTC, the user first sends a Stacks trnsaction to `.sbtc`
+to indicate the following:
+
+* How much sBTC they would like to redeem
+
+* The Bitcoin address to which the BTC will be sent
+
+* As an optional hint, the list of UTXOs that could be consumed to produce a BTC
+  redemption transaction
+
+This request is queued up in the `.sbtc` data space, which Stackers (through
+their signer daemons) observe and react to.  When the transaction confirms, the
+user's requested sBTC is locked until the end of the next reward cycle -- the
+user cannot spend it while Stackers are considering the request.
+
+![Nakamoto Flow Diagrams (1)](https://github.com/stacksgov/sips/assets/459947/e8da8826-1303-4891-83fb-a568ec9d3f9d)
+
+_Figure 2: Protocol diagram for a successful peg-out request.  The user submits
+a peg-out request to Stacks, which locks their requested sBTC tokens until the
+start of the next reward cycle.  If the peg-out succeeds, then they receive BTC from
+the Stackers (minus a BTC transaction fee), and their sBTC gets burnt._
+
+When Stackers see the peg-out request, and if they are able to act on it (i.e.
+the wallet has enough UTXOs free), they execute a WSTS signing round to
+generate a Bitcoin transaction from the peg wallet's UTXOs which pays out to the
+user's requested address.  Because all pegged-in BTC is spendable by the current
+set of Stackers via an aggregate Schnorr signature, Stackers simply create the
+scriptSigs for sufficiently many UTXOs to fund a fulfillment UTXO for the user.
+
+Because fulfilling the redemption request occurs through WSTS, there is a
+possibility of a quorum failure.  As such, the Stackers may instead vote to
+abstain or even to refuse the peg-out request.  This is explained in the
+"Failure Modes" section.
+
+The BTC redemption transaction contains the following fields:
+
+* An `OP_RETURN` identifying the peg-out request in the `.sbtc` contract
+
+* A new UTXO that redeems the user's sBTC for BTC
+
+* A change UTXO that consolidates the remaining unused BTC from the consumed
+  wallet UTXOs
+
+Once the Stackers have created the BTC redemption transaction, they both send it
```

Comment:
[nit] "they both sending it ..." was confusing to read. Removing "both" or moving it after send seems clearer? something like "they broadcast it to both a Bitcoin peer and to the Stacks network"

Reply:
Sure -- will remove